src/agentscope/agents/dialog_agent.py

```diff
@@ -31,15 +33,16 @@ def __init__(
                 configuration.
             use_memory (`bool`, defaults to `True`):
                 Whether the agent has memory.
-            memory_config (`Optional[dict]`):
-                The config of memory.
         """
         super().__init__(
             name=name,
             sys_prompt=sys_prompt,
             model_config_name=model_config_name,
             use_memory=use_memory,
-            memory_config=memory_config,
+        )
+
+        logger.warning(
+            f"Unused keyword arguments is provided: {kwargs}",
```

Comment:
should we check if kwargs exists here? Otherwise, it will always log out
"Unused keyword arguments is provided: {}"

Reply:
Changed in the new commit.